samples/betaFeatures.js

```diff
@@ -0,0 +1,547 @@
+    diarizationSpeakerCount: 2,
+    model: `phone_call`,
+  };
+
+  const audio = {
+    content: fs.readFileSync(fileName).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+      console.log(`Speaker Diarization:`);
+      const result = response.results[response.results.length - 1];
+      const wordsInfo = result.alternatives[0].words;
+      // Note: The transcript within each result is separate and sequential per result.
+      // However, the words list within an alternative includes all the words
+      // from all the results thus far. Thus, to get all the words with speaker
+      // tags, you only have to take the words list from the last result:
+      wordsInfo.forEach(a =>
+        console.log(` word: ${a.word}, speakerTag: ${a.speakerTag}`)
+      );
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_diarization]
+}
+
+function asyncSpeechTranscribeDiarizationGCS(gcsUri) {
+  // [START speech_transcribe_diarization_gcs]
+
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const uri = path to GCS audio file e.g. `gs:/bucket/audio.wav`;
+
+  const config = {
+    encoding: `LINEAR16`,
+    sampleRateHertz: 8000,
+    languageCode: `en-US`,
+    enableSpeakerDiarization: true,
+    diarizationSpeakerCount: 2,
+    model: `phone_call`,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+      console.log(`Speaker Diarization:`);
+      const result = response.results[response.results.length - 1];
+      const wordsInfo = result.alternatives[0].words;
+      // Note: The transcript within each result is separate and sequential per result.
+      // However, the words list within an alternative includes all the words
+      // from all the results thus far. Thus, to get all the words with speaker
+      // tags, you only have to take the words list from the last result:
+      wordsInfo.forEach(a =>
+        console.log(` word: ${a.word}, speakerTag: ${a.speakerTag}`)
+      );
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_diarization_gcs]
+}
+
+function speechTranscribeMultiChannel(fileName) {
```

Comment:
Please re-add `TranscribeMultiChannelGCS` sample - you just had a typo.

Reply:
+1 (@CallistoCF I added this back to the description region tag list)

Reply:
( **also:** *Hiiiiiii* @jerjou!!!! 👋👋👋 )

Reply:
I still can't seem to get this to work, tried long running - I'm just getting the single audio response here, any idea why this isn't working?  As Python canonical doesn't have this, I have this commented out at the moment-

Reply:
It worked for me after I capitalized the `p` in `Per` (in `enableSeparateRecognitionPerChannel`). Would you mind pushing your latest, so I can see if it works for me?

(hi rebecca! I didn't recognize you there :-)

Reply:
That did it!  i should have test in just a couple minutes...